Engineering/Domain model in Domain Driven Design.md

```diff
@@ -0,0 +1,38 @@
+---
+tags: engineering, domain-driven-design, DDD
+author: M.Vu Cuong(Jim)
+---
+
+### Recap on Domain Driven Design
+
+A software design approach to create a rich and meaningful **Domain model** which enforces the conversation around the domain. As result, removes _the communication lag_ between **Developers** and **Domain/Business experts**.
+
+[Here is the full overview](obsidian://open?vault=brain&file=Engineering%2FOverview%20of%20Domain%20Driven%20Design)
```

Comment:
I think including the recap is kind of unnecessary. Maybe just mention the DDD article from the introduction of Domain model. e.g:

```
In [Domain Driven Design](link), Domain Model is ....
```